dagmc_stats.py

```diff
@@ -1,25 +1,24 @@
 # set the path to find the current installation of pyMOAB
 import sys
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
+import numpy as np
```

Comment:
`numpy` isn’t used here - it’s best to avoid introducing unnecessary dependencies